wpilibc/src/main/native/cpp/SpeedControllerGroup.cpp

```diff
@@ -43,9 +43,7 @@ void SpeedControllerGroup::StopMotor() {
 }
 
 void SpeedControllerGroup::PIDWrite(double output) {
```

Comment:
wpiformat on Travis failed. You can either set up wpiformat to fix it for you, or look at the diff in the Travis build log and manually apply the changes.